python/tank/descriptor/descriptor_bundle.py

```diff
@@ -56,6 +60,124 @@ def version_constraints(self):
 
         return constraints
 
+    _sg_studio_versions = {}
+
+    @classmethod
+    def _get_sg_version(cls, connection):
+        """
+        Returns the version of the studio shotgun server. It caches the result per site.
+
+        :param connection: Connection to the Shotgun site.
+        :type: :class:`shotgun_api3.Shotgun`
+
+        :returns: a string on the form "X.Y.Z"
+        :rtype: str
+        """
+        if connection.base_url not in cls._sg_studio_versions:
+            try:
+                version_tuple = connection.server_info["version"]
```

Comment:
Good point! Just looked at the code and seems the value is cached now. I had assumed that since there was a cache in Toolkit before for that value that we still needed one today. Thanks for having me look.